middlewares/validators/search.validator.js

```diff
@@ -11,4 +11,24 @@ module.exports = {
         VALIDATOR.booleanValidator("query", "expand", true),
         VALIDATOR.searchValidator("query", "q")
     ],
+    statusValidator: [
+        VALIDATOR.searchModelValidator("query", "model", false),
+        VALIDATOR.alphaValidator("query", "sort", true),
+        VALIDATOR.integerValidator("query", "page", true, 0),
+        VALIDATOR.integerValidator("query", "limit", true, 0, 1000),
+        VALIDATOR.searchSortValidator("query", "sort_by"),
+        VALIDATOR.booleanValidator("query", "expand", true),
+        VALIDATOR.searchValidator("query", "q"),
+        VALIDATOR.updateHackerValidator("query", "update")
+    ],
+    emailValidator: [
```

Comment:
Same notes as above, but also, it's kinda weird that the emailValidator checks the status? I see this is being used in the bulk email endpoint, but should the name of a thing describe what it does, or where it's being used? What kind of coupling does that cause?